bin/plugin/commands/performance.js

```diff
@@ -95,23 +113,25 @@ async function runPerformanceTests( branches, options ) {
 		branches = [ 'trunk' ];
 	}
 
+	log( fTitle( '\n💃 Performance Tests 🕺' ) );
 	log(
-		formats.title( '\n💃 Performance Tests 🕺\n' ),
-		'\nWelcome! This tool runs the performance tests on multiple branches and displays a comparison table.\n' +
-			'In order to run the tests, the tool is going to load a WordPress environment on ports 8888 and 8889.\n' +
-			'Make sure these ports are not used before continuing.\n'
+		'\nWelcome! This tool runs the performance tests on multiple branches and displays a comparison table.'
 	);
 
 	if ( ! runningInCI ) {
+		log(
+			fWarning(
+				[
+					'\nIn order to run the tests, the tool is going to load a WordPress environment on ports 8888 and 8889.',
+					'Make sure these ports are not used before continuing.\n',
+				].join( '\n' )
+			)
+		);
+
 		await askForConfirmation( 'Ready to go? ' );
 	}
 
-	/*
-	 * 1- Preparing the tests directory.
-	 */
-
-	log( '\n>> Preparing the tests directories' );
-	log( '    >> Cloning the repository' );
+	logStep( 0, 'Setting up' );
```

Comment:
okay so is the `sub` supposed to connote some kind of level of depth of sub-step?

the `0` here confused me when I came across it. I wonder if you could accomplish what you're looking for by creating a new logging function and setting up a parent/child relationship with that, or of providing a method on the function itself to communicate indent depth

```js
logStep( 'Setting up' );
logStep.indented( 1 ).log( 'Removing existing files' );

logSubstep = logStep.indented( 1 );
logSubstep( 'Removing existing files' );
```

Reply:
or even naming this something like `logAtDepth( 2, 'Creating directory:', … )` could help. or `logAtIndent( 2, 'Creating…' )`